src/Workspaces/CSharp/Portable/Rename/CSharpRenameRewriterLanguageService.cs

```diff
@@ -550,14 +550,17 @@ private SyntaxToken RenameToken(SyntaxToken oldToken, SyntaxToken newToken, stri
                         }
                     }
                 }
-                else if (isAccessorLocation)
-                {
-                    var prefix = oldIdentifier.Substring(0, oldIdentifier.IndexOf('_') + 1);
-                    currentNewIdentifier = prefix + currentNewIdentifier;
-                }
-                else if (!string.IsNullOrEmpty(suffix))
+                else
                 {
-                    currentNewIdentifier = _replacementText + suffix;
+                    if (!string.IsNullOrEmpty(prefix))
```

Comment:
No difference. Just decided against pulling on that string as part of this change. I'd be happy to clean it up though.